src/commands.py

```diff
@@ -260,9 +260,16 @@ def cmd_transaction(ep, c):
     else:
         outputBackChange = None
 
+    tx = list()
+    tx.append(["amount (" + get_current_block(ep)["currency"] + ")", amount / 100])
+    tx.append(["amount (DU " + get_current_block(ep)["currency"] + ")", amount / du])
```

Comment:
can we stock the value of of the current_block in variable to avoid multiples request on the network?
but i think the best way is to change the function "get_current_block" to take only one request on the network for the multiples call.